viYarn/Episode/view/EpisodeViewController.swift

```diff
@@ -8,14 +8,24 @@
 
 import UIKit
 
-class EpisodeTableViewController: UITableViewController {
+class EpisodeViewController: UIViewController, EpisodeLoaderDelagate {
+    
+    lazy var tableView: UITableView = {
+        let table:UITableView = UITableView()
+        table.delegate = viewModel
+        table.dataSource = viewModel
+        table.translatesAutoresizingMaskIntoConstraints = false
+        return table
+    }()
+    
+    var viewModel = MockEpisodeViewModel()
```

Comment:
p0: make it a constant and put it into the init like this:
init(viewModel: EpisodeViewModelResponsibilities) {
self.viewModel = viewModel
super.init(nil, nil)
}